### DISCUSSION TITLE
Add Stable's combo counters from mania, taiko and CTB

### DISCUSSION THREAD

**Discussion by Corne2Plum3** [ppy/osu#22901] [Ideas]
In stable, each osu! game mode uses a different combo counter (maybe they looks the same, but have different animations).

For exemple, in Mania, there's this combo counter, which doesn't have a "x", and is centered, unlike the one in stable where the text is aligned to the bottom left. It would be cool to see this component on lazer, to recreate even more accurately the experience on stable.

![](https://i.imgur.com/BgM2RJ0.jpg)

On taiko and CTB it's the same combo counter but with different animations when the combo increases.

**Comment by Axilotl17** [comment]
I think this will be available as skinning improves, although it might be cool to implement this into argon.

**Comment by peppy** [comment]
The skinning framework required for this was added just last release (#22673). Expect to see this in the near future.
